src/server/accessors/UserRead.ts

```diff
@@ -13,7 +13,11 @@ export class UserRead implements IUserRead {
         return this.userBridge.getByUsername(username, this.appId);
     }
 
-    public getAppUser(appId: string): Promise<IUser | undefined> {
+    public getAppUser(appId?: string): Promise<IUser | undefined> {
```

Comment:
Use [default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) here

Reply:
Good suggestion! I've just implemented it.